package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ramda.github.io",
-  "version": "0.0.0",
+  "version": "0.19.1",
```

Comment:
seems a separate concern to me. if we make some major change to how the docs are built/presented, we should be able to increment this value independently of the lib. otoh, this package.json is not for actual distribution of a package anywhere.